timeseries/tests/unittests/models/test_sktime.py

```diff
@@ -5,21 +5,10 @@
 import pytest
 
 from autogluon.timeseries import TimeSeriesDataFrame
-
```

Comment:
To clarify, we're removing this check here since we will anyway import the Sktime models in `timeseries.models.sktime.models` and we would get a failure if that import didn't work, right?

Reply:
Hmm, you're right this seems to have been the intention; but now we're treating sktime as an optional dependency in the `setup.py` but a hard dependency in the code. We should fix this in migrating to statsmodels, adding it as a core dependency?